### PR TITLE
Update BEP20Token.template

### PR DESCRIPTION
**Pull request by RumeelHussainbnb**
The solidity compiler is outdated. The code is updated to match newer compiler versions.

### CODE REVIEW

**Review by hamzawajahat — APPROVED** [review]

**Comment by guagualvcha** [comment]
hi, @RumeelHussainbnb  if you are looking for the standard BEP20 contract, it is https://github.com/bnb-chain/canonical-upgradeable-bep20.  The contract here is a quick demo used by other genesis contracts.

**Comment by hamzawajahat** [comment]
Awww thanks

**Review by Pythonberg1997 — APPROVED** [review]

**Review by guagualvcha — APPROVED** [review]

File: contracts/bep20_template/BEP20Token.template [hunk 1]
```diff
@@ -1,4 +1,5 @@
-pragma solidity 0.5.16;
```
Comment (hamzawajahat) [inline contracts/bep20_template/BEP20Token.template:1]: Update it

**Comment by shango420** [comment]
https://data-seed-prebsc-1-s1.bnbchain.org:8545

**Pull request merged**